src/sidebar.js

```diff
@@ -163,7 +163,12 @@ module.exports = {
         {
           type: "category",
           label: "Passwordless",
-          items: ["kratos/passwordless/webauthn"],
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "kratos/passwordless",
```

Comment:
I propose to:
- move Passwordless above Social Sign-In?
- rename Apple Passkeys to "Passkeys & Biometric" and move above WebauthN

Reply:
Sure, I can move it above social sign-in. I don't think "Passkeys & Biometric" is a good name though. When using passkeys, any biometrics that the user has to "supply" are required to confirm that they want to use the passkey. The passkey itself is just cryptography.

Reply:
Also, passkeys don't require biometrics. If a device is not biometrics-enabled, users can still use passkeys.